shopinvader_delivery_carrier_category_keep_carrier/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Shopinvader Delivery Carrier Category Keep Carrier",
+    "summary": """
+        Allows to keep carrier even on delivery address change""",
```

Comment:
Where is the `readme` directory? Can you provide at least a description on how this addon works? At least, we should find into the description that this addon allows you to specify on a delivery carrier category if the carrier information on the SO must be reset when the shipping address change.